docs/source/tutorials/faq.ipynb

```diff
@@ -68,16 +68,26 @@
    "id": "3868ee8b",
    "metadata": {},
    "source": [
-    "### Why isn’t CleanLearning working for my data?"
+    "### Why isn’t CleanLearning working for me?"
    ]
   },
   {
    "cell_type": "markdown",
    "id": "d13c9cd0",
    "metadata": {},
    "source": [
-    "At this time, CleanLearning only works directly with numpy matrices or pd.DataFrames.\n",
-    "You can still use cleanlab with other data formats though! Just separately obtain predicted probabilities from your model via cross-validation and pass them as inputs."
+    "At this time, CleanLearning only works with data formatted as numpy matrices or pd.DataFrames, \n",
+    "and with models that are compatible with the `sklearn` API.\n",
+    "You can still use cleanlab with other data formats though! Just separately obtain predicted probabilities (`pred_probs`) from your model via cross-validation and pass them as inputs.\n",
+    "\n",
+    "\n",
+    "If CleanLearning is running successfully but not improving predictive accuracy of your model, here are some tips:\n",
+    "\n",
+    "1. Use cleanlab to find label issues in your test data as well (we recommend pooling `labels` across both training and test data into one input for `find_label_issues()`). Then manually review and fix label issues identified in the test data to verify accuracy measurements are actually meaningful.\n",
+    "\n",
+    "2. Try different values for `filter_by`, frac_noise`, and `min_examples_per_class` which can be set via the `find_label_issues_kwargs` argument in the initialization of `CleanLearning()`. \n",
+    "\n",
+    "3. Try to find a better model (eg. via hyperparameter tuning or changing to another classifier). `CleanLearning` can find better label issues by leveraging a better model, which allows it to produce better quality training data. This can form a virtuous cycle in which better models -> better issue detection -> better data -> even better models!"
```

Comment:
all good. here are some others.

4. Try combining (using AND operator in boolean space) any subset of the filters for more restrictive filtering.
5. Try jointly hyper-parameter tuning both the model and filter_by parameter simultaneously
6. Does your dataset have 'clutter' class or a 'junk' class? If you have bad data, consider creating one (e.g. Caltech-256)
7. Try merging similar/overlapping classes for significantly better performance (e.g. see this tutorial: https://docs.cleanlab.ai/v2.0.0/tutorials/indepth_overview.html#Workflow-3:-Use-cleanlab-to-find-dataset-level-and-class-level-issues)
8. If your `pred_probs` are obtained via a neural network, try averaging the `pred_probs` over the last K epochs of training instead of just using the final `pred_probs`. Similarly, you can try averaging `pred_probs` from several models (remember to re-normalize).

Reply:
I don't think 4 or 8 are specific to CleanLearning?  Trying to keep this question just about CleanLearning, feel free to add a more general tips section for finding better label issues in separate PR

Reply:
added the other points

Reply:
why not just add 4 and 8 at the bottom as its own section "General Tips that might also help improve model accuracy"? Can create a new FAQ later, but adding them there now can't hurt?

Reply:
Sure I'm onboard with that, can you add them?  I won't edit this PR anymore.